### PR TITLE
Add auth & app password docs and clean up README

### PR DESCRIPTION
**Pull request by emilyliu7321**
- Adds auth documentation and details for app passwords
- Tidy up README

### CODE REVIEW

File: content/specs/atp.md [hunk 1]
```diff
@@ -256,7 +256,20 @@ The client-to-server API drives communication between a client application and t
 
 ### Authentication
 
-<div class="todo">Describe how the client authenticates with the PDS. (It's a simple JWT-based session.)</div>
+Authentication is a simple session-oriented process. View the API call in the applications model section of the docs [here](https://atproto.com/guides/applications#signing-in).
+
+#### App passwords
+We also have app passwords, a short-term solution for authentication that will let users experiment with third-party clients without having to fully trust them with their passwords. In the long term, we plan to implement SSO (Single Sign-On) with scoped permissions.
```
Comment (Jacob2161) [inline content/specs/atp.md:262]:
```suggestion
We also have app passwords, an initial solution for authentication that will let users use third-party clients without having to trust them with their primary password. In the long term, we plan add SSO (Single Sign-On) authentication with scoped permissions.
```

File: content/specs/atp.md [hunk 2]
```diff
@@ -256,7 +256,20 @@ The client-to-server API drives communication between a client application and t
...
+
+Users can log into third-party apps with an app password in the same way that they login with their account password. App passwords have most of the same abilities as the user's account password, but they're restricted from destructive actions such as account deletion or account migration. They are also restricted from creating additional app passwords.
+
+No client changes are required to adopt app passwords. However, we strongly encourage you to prompt users to use an app password on login and avoid ever entering their password. For account creation, we encourage redirecting a user to the bluesky client (https://staging.bsky.app).
```
Comment (Jacob2161) [inline content/specs/atp.md:266]:
```suggestion
No client changes are required to adopt app passwords. However, we strongly encourage you to prompt users to use an app password on login and avoid ever entering their password. For account creation, we encourage redirecting a user to the [Bluesky web client](https://staging.bsky.app).
```

**Review by Jacob2161 — COMMENTED** [review]
Nice! Added some minor suggestions, the main one intended to avoid implying that app passwords are temporary since 1. it might spook devs and 2. they're probably not going anywhere soon.

**Comment by emilyliu7321** [comment]
@Jacob2161 committed both of your edits 🙏

File: content/specs/atp.md [hunk 3]
```diff
@@ -256,7 +256,20 @@ The client-to-server API drives communication between a client application and t
...
+
+#### App passwords
+We also have app passwords, an initial solution for authentication that will let users use third-party clients without having to trust them with their primary password. In the long term, we plan add SSO (Single Sign-On) authentication with scoped permissions.
+
+Users can log into third-party apps with an app password in the same way that they login with their account password. App passwords have most of the same abilities as the user's account password, but they're restricted from destructive actions such as account deletion or account migration. They are also restricted from creating additional app passwords.
+
+No client changes are required to adopt app passwords. However, we strongly encourage you to prompt users to use an app password on login and avoid ever entering their password. For account creation, we encourage redirecting a user to the [Bluesky web client](https://staging.bsky.app).
+
+We also strongly encourage you to delete all existing refresh tokens and re-fetch access/refresh tokens using an app password.
```
Comment (devinivy) [inline content/specs/atp.md:268]: It took me a second to parse this.  I think was I was missing was just the context that this is for clients who are migrating from accepting primary passwords to app passwords.  Maybe a small tweak like,
```suggestion
If you expect users have used their primary password with your client, we also strongly encourage you to delete all existing refresh tokens and re-fetch access/refresh tokens using an app password.
```

**Review by devinivy — APPROVED** [review]
This is great!

**Pull request merged**